README.md

```diff
@@ -1 +1,11 @@
-# fruit-salad-team-6
\ No newline at end of file
+
+project-name:fruit-salad.
+
+TEAM leader: sara abu awwad.
+TEAM Members: Rawan Zaqout , Mohammed Hasan , Safa Mohanna.
```

Comment:
no need to add my name 

also before the (team member) add `##` as a second heading, and add your names as a list, it will look like that: 

## Team Member 

- Rawan 
- Mohammed
- Safa